src/components/templates/ExternalRoom/ExternalRoom.tsx

```diff
@@ -67,7 +67,12 @@ export const ExternalRoom: React.FC<ExternalRoomProps> = ({ venue }) => {
         <>
           <div className="ExternalRoom__message">
             <div>This page should automatically open</div>
-            <a rel="noreferrer" href={redirectUrl} target="_blank">
+            <a
+              rel="noreferrer"
+              href={redirectUrl}
+              target="_blank"
+              className="ExternalRoom__link"
+            >
```

Comment:
```suggestion
              href={redirectUrl}
              className="ExternalRoom__link"
              {...getExtraLinkProps(true)}
            >
```

Reply:
nice catch! fixed in 0ee2698825ee60395a850a7d52819eb590cb92c8